frontend/src/actions/projectActions.ts

```diff
@@ -12,7 +12,7 @@ import {TimesheetCheckAttachmentType} from '../models';
 import {authService} from '../components/users/authService';
 import {FullProjectMonthModel} from '../components/project/models/FullProjectMonthModel';
 
-export function saveProject(project: IProjectModel, navigate?: any, after: 'to-list' | 'to-details' = 'to-list') {
+export function saveProject(project: IProjectModel, navigate?: any, after: 'to-list' | 'to-details' = 'to-list', triggerCopy?: any) {
```

Comment:
Probeer altijd zo typed mogelijk te zijn

vb:
```
triggerCopy: (projectId: string) => void;
```

Ik ben wel effe aan het bekijken of het misschien makkelijker kan...